starkjs/README.md

```diff
@@ -5,53 +5,78 @@ PIL compiler and Circom transpiler. The stark prover is [starky](../starky).
 ## Run Example
 ### Arithmetization: Constraint Polynomial
 
-```
+```bash
 export CIRCUIT=fib
 npm run $CIRCUIT
 ```
 will generate the PIL json, Commitment Polynomial file and Constant Polynomial file.
 
 ### Bottom Layer: FRI Proof
 
-```
+```bash
+export CIRCUIT=fib
 ../target/release/eigen-zkit stark_prove -s ../starky/data/starkStruct.json.gl \
     -p /tmp/$CIRCUIT.pil.json \
     --o /tmp/$CIRCUIT.const \
     --m /tmp/$CIRCUIT.cm -c circuits/$CIRCUIT.verifier.circom --i circuits/$CIRCUIT.verifier.zkin.json
 ```
 
 ### Recursive Layer: FRI Proof
+test compressor12
+#### old script
+```bash
+export CIRCUIT=fib
+#../target/release/eigen-zkit compile -p goldilocks -i circuits/$CIRCUIT.verifier.circom -l node_modules/pil-stark/circuits.gl --O2=full -o /tmp/
 
-```
-../target/release/eigen-zkit compile -p goldilocks -i circuits/$CIRCUIT.verifier.circom -l node_modules/pil-stark/circuits.gl --O2=full -o /tmp/
-
-// Circom to Stark  
-node src/compressor12/main_compressor12_setup.js \
+# Circom to Stark  
+time node src/compressor12/main_compressor12_setup.js \
     -r /tmp/$CIRCUIT.verifier.r1cs \
-    -c /tmp/c12.const \
-    -p /tmp/c12.pil \
-    -e /tmp/c12.exec
+    -c /tmp/c12_old.const \
```

Comment:
We'll remove the js compressor12 directory, so the commands in this file also should be removed/replaced